components/family-studies/ui/src/main/resources/PhenoTips/FamilySheet.xml

```diff
@@ -97,8 +97,11 @@
 ##
 ## Extensions displayed after the content
 #foreach ($e in $services.uix.getExtensions('phenotips.familyRecord.after', {'sortByParameter' : 'order'}))
+  #set ($extension_disabled = ($e.getParameters().get('enabled') == 'false'))
+  #if (!$extension_disabled)
+    {{html clean=false}}$services.rendering.render($e.execute(), 'xhtml/1.0'){{/html}}
```

Comment:
Rather _unfortunately_. If we had had any, I would have noticed the problem. Thanks!